sdk/python/kfp/compiler/compiler.py

```diff
@@ -714,51 +714,87 @@ def _sanitize_and_inject_artifact(self, pipeline: dsl.Pipeline):
       sanitized_ops[sanitized_name] = op
     pipeline.ops = sanitized_ops
 
-  def _compile(self, pipeline_func):
-    """Compile the given pipeline function into workflow."""
-    # Step 1: extract param value, name and description from pipeline_func signature and decoration.
+  def create_workflow_from_func(self,
```

Comment:
Can we create workflow from something other than func? If not, maybe consider renaming to `create_workflow` ?

Reply:
Indeed. Done.